jetty-client/src/main/java/org/eclipse/jetty/client/MultiplexConnectionPool.java

```diff
@@ -64,7 +64,11 @@ public Connection acquire()
         Connection connection = activate();
         if (connection == null)
         {
-            int maxPending = 1 + destination.getQueuedRequestCount() / getMaxMultiplex();
```

Comment:
How about this one-liner instead?
```
int maxPending = (destination.getQueuedRequestCount() + getMaxMultiplex() - 1) / getMaxMultiplex();
```

Reply:
That is obfuscated... Does it do `ceiling(a/b)`?

Reply:
Yes, it's the integer version of the following pseudo-code: `Math.ceil(destination.getQueuedRequestCount() / getMaxMultiplex())`